src/plain_model.rs

```diff
@@ -0,0 +1,411 @@
+//! Non-generated version of required structures provided by the protobuf.
+//! This version is used when the `protobuf` feature is turned off.
+
+#![allow(missing_docs)]
+
+use std::default::Default;
+
+#[derive(PartialEq, Clone, Default, Debug)]
+pub struct LabelPair {
+    name: String,
+    value: String,
+}
+
+impl LabelPair {
+    #[deprecated(note = "Use default()", since = "0.5.1")]
```

Comment:
I don't think this is necessary. It's less "deprecated" and more "the function signature may change over time"

Reply:
Hm, I don't think `new()`'s signature would change if we kept it? 

I deprecated all the `new()` because all they do is call `default()` which is redundant. I wanted to get rid of to protobuf-isms and take the API closer to what it would be like if it hadn't been generated.

Reply:
I'm ok with not deprecating stuff, this can be done later. What would be your suggestion?